src/views/project/ProjectDetails.vue

```diff
@@ -14,18 +14,14 @@
 					v-col(md="6")
 						div(class="float-right")
 							CreateUpdateProject(update :project="Object.assign({}, project)" v-bind:getProjectDetails="getProjectDetails")
-							v-btn(tile depressed color="teal" large class="ma-2" dark)
+							v-btn(tile depressed color="teal" large class="ma-2" dark v-bind:href="`/repository?project_id=${this.$route.params.id}`")
 								v-icon(left) mdi-plus
 								span Add text
```

Comment:
I could not add a text. I got OSError: b'{"detail":"Invalid token header. Token string should not contain spaces."}'

Reply:
Fixed by https://github.com/diging/vogon-web/pull/152